lib/dialects/mysql.ts

```diff
@@ -0,0 +1,123 @@
+import Knex from 'knex';
+import { SchemaInspector } from '../types/schema-inspector';
+import { MySQLTable } from '../types/table';
+import { MySQLColumn } from '../types/column';
+
+export default class MySQL implements SchemaInspector {
```

Comment:
we need an integration test for this. docker setup could be copied from knex, see https://github.com/knex/knex/blob/master/scripts/docker-compose.yml

Reply:
Yes! I wanted to talk to you about setting up a testing environment, I'm not too familiar with how to set that up with all these different database abstractions

Reply:
sure, I can set it up together with CI then